package.json

```diff
@@ -1,28 +1,39 @@
 {
-  "name" : "memory-cache",
-  "description" : "A simple in-memory cache. put(), get() and del()",
-  "author" : "Paul Tarjan <npm@paulisageek.com>",
+  "name": "memory-cache",
+  "description": "A simple in-memory cache. put(), get() and del()",
+  "author": "Paul Tarjan <npm@paulisageek.com>",
   "contributors": [
     {
       "name": "Ramon Snir",
       "email": "ramon@dynamicyield.com"
     }
   ],
-  "url" : "https://github.com/ramonsnir/node-cache",
-  "keywords" : [
+  "url": "https://github.com/ramonsnir/node-cache",
+  "keywords": [
     "cache",
     "ram",
     "simple",
     "storage"
   ],
-  "main" : "./index.js",
-  "version" : "0.1.1",
-  "repository" : { 
-    "type" : "git", 
-    "url" : "git://github.com/ptarjan/node-cache.git"
+  "main": "./index.js",
+  "version": "0.1.1",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ptarjan/node-cache.git"
   },
-  "scripts" : {
-    "test" : "node test.js"
+  "scripts": {
+    "test": "node test.js"
   },
-  "license" : "BSD"
+  "license": "BSD",
+  "devDependencies": {
```

Comment:
The only real diff in this file is with the `devDependencies`. `npm` caused all the whitespace changes automatically when I added the `devDependencies`.